pkg/network/protocols/http/shared_libraries.go

```diff
@@ -35,118 +37,213 @@ func (l *libPath) Bytes() []byte {
 	return l.Buf[:l.Len]
 }
 
-// syncInterval controls the frequency at which /proc/<PID>/maps are inspected.
-// this is to ensure that we remove/deregister the shared libraries that are no
-// longer mapped into memory.
-const soSyncInterval = 5 * time.Minute
+// pathIdentifier is the unique key (system wide) based on dev/inode
+type pathIdentifier struct {
+	dev   uint64
+	inode uint64
+}
+
+func (p *pathIdentifier) String() string {
+	return fmt.Sprintf("dev/inode %d.%d/%d", unix.Major(p.dev), unix.Minor(p.dev), p.inode)
+}
+
+func (p *pathIdentifier) Key() string {
+	b := make([]byte, 16)
+	binary.LittleEndian.PutUint64(b, p.dev)
+	binary.LittleEndian.PutUint64(b[8:], p.inode)
+	m := murmur3.Sum64(b)
+	bsum := make([]byte, 8)
+	binary.LittleEndian.PutUint64(bsum, m)
+	return base64.StdEncoding.EncodeToString(bsum)
```

Comment:
please use meaningful names for the variables

Reply:
done